### PR TITLE
Gem sim validation porting 76 x only occupy plots

### PR DESCRIPTION
**Pull request by geonmo**
### Hit Validation module for GEM Detector after Run2
1. This PR include Validation/MuonGEMHits, MuonGEMDigis, MuonGEMRecHits module to validate GEM hit simulation.
   - MuonGEMHits module is related to SimHit on GEM Detector(Run2019/2023).
   - MuonGEMDigis module is related to Digi step about GEM Strip, pad and Copad.
   - MuonGEMRechits module is related to local reco hit on GEM Detector. 
#### This PR does not affect to running other modules.
- These module are only running with specific customize options.

Remake PR due to wrong push to my branch and too old.
It is same PR #11123 except some fix as follow comments.

### CODE REVIEW

**Comment by geonmo** [comment]
@deguio Hello, Federico. I re-make a PR due to mistake on previous PR. Could you test this?

**Comment by deguio** [comment]
please test

**Comment by geonmo** [comment]
@deguio Hello, Federico. Could you review for this PR?

**Comment by deguio** [comment]
+1
detailed presentation at:
https://indico.cern.ch/event/446270/contribution/2/attachments/1154267/1658925/Presentation_on_DQM_MeetingGEM_Validation_modules.pdf

File: Validation/MuonGEMDigis/interface/GEMCheckGeometry.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+
+#include "Validation/MuonGEMHits/interface/GEMBaseValidation.h"
+#include "Geometry/CommonTopologies/interface/StripTopology.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/GEMDigi/interface/GEMDigiCollection.h"
+#include <TMath.h>
```
Comment (davidlange6) [inline Validation/MuonGEMDigis/interface/GEMCheckGeometry.h]: most of these header files are either not needed or should be in the c++.. could you try to optimize this a bit?

**Comment by geonmo** [comment]
@davidlange6 Hello, david! I am sorry about late updates. I am so busy to prepare our regional conference. Could you review my code about head optimization?

**Comment by geonmo** [comment]
Well, I re-checked a PR before push using "Comparision between base and head".
I think that it is not correctly working as my guess
Ok, I will remake a new PR with same contents.

**Comment by geonmo** [comment]
@cmsbuild, @cmsbuild, @danduggan, @vanbesien, @deguio and @davidlange6 
Hello, everyone. I remake a new PR with same contents. Please, see PR#11948 for this issue.
Thanks

**Pull request closed without merging**